docs/Configuration.md

```diff
@@ -55,11 +55,12 @@ description | Description of the group | optional
 path | Path of the intermediate node either relative or absolute. If relative, `/home/groups` is automatically prefixed. By default some implementation specific path is choosen. Usually the full group path is the (intermediate) path concatenated with a [randomized authorizable id](https://jackrabbit.apache.org/oak/docs/apidocs/org/apache/jackrabbit/oak/security/user/RandomAuthorizableNodeName.html).  | optional
 isMemberOf | List of groups this groups is a member of. May be provided as yaml list or as comma-separated yaml string (*the use of comma-separated yaml strings is deprecated*, available to remain backwards compatible). | optional
-memberOf | same meaning as `isMemberOf`. This property is *deprecated*, please use `isMemberOf` instead. | optional
+memberOf | Same meaning as `isMemberOf`. This property is *deprecated*, please use `isMemberOf` instead. | optional
 members | List of groups that are member of this group. May be provided as yaml list or as comma-separated yaml string (*the use of comma-separated yaml strings is deprecated*, available to remain backwards compatible). Allows to specify the relationship from the other side compared to `isMemberOf`. **NOTE**: The `isMemberOf` configuration should be preferred over `members` if possible for better readability. Since `isMemberOf` shows what a group inherits, `members` would only show on the other side where permissions get pushed to. Also choosing one approach makes the config files consistent. `members` should therefore only be used in special cases where it is not possible to use `isMemberOf`, e.g. to push rights to a group that is not in the configuration and hence that group can not be annotated with `isMemberOf` | optional
-migrateFrom | a group name assigned member users are taken over from | optional
+migrateFrom | A group name assigned member users are taken over from. This is not supported for external user management. | optional
 virtual | If true, the group is *not* created in repository but only its permissions and isMemberOf relationships are made effective to other groups using the virtual group in isMemberOf. Flattens the effective group tree of users at runtime, but enlarges ACLs as set on path (instead of the virtual group, all referencing groups are listed in a particular path's ACL used by the virtual group). Useful for functionality fragments. False by default. Since v2.1.0 | optional
 unmanaged* Properties | Only use sparsely and with care, see [Advanced Features](AdvancedFeatures.md)   | optional
+syncWithExternalUserManagement | Boolean flag determining whether the group should also be created or updated in a configured external user management system. For further details refer to [Update Groups in External User Management Systems](AdvancedFeatures.md#Update Groups in External User Management Systems). Since v3.1.0. | optional
```

Comment:
so I don't like the user management in this term  (for the reason stated in previous comment). Either `syncWithExternalGroupManagement` or maybe even better `ensureCreatedInIdP`. The latter would also allow to use this feature potentially on publish (on publish it could check for an implementation of a provider interface provided by a project to create the marked groups anywhere needed)

Reply:
see my comments before, the idea was to have a flag which could be used both in groups and users (although right now is only supported in groups). For me the most important aspect of this flag is `external`.

Reply:
At the moment we are missing a full example, currently a group definition would look as follows:

```
    - example-group:
        - name: "Example Group"
          externalId: "example-group:ims"
          path: "/home/groups/example"
```
maybe the most straight forward, simple and concise way of configuring it is the following:

```
    - example-group:
        - name: "Example Group"
          externalId: "example-group:ims"
          externalSync: true
          path: "/home/groups/example"
```

Reply:
I don't think a full example is necessary with the description of both fields `externalSync` and `externalId` now, right? This is anyhow not the place where to put such an example...